cgnsutilities/cgnsutilities.py

```diff
@@ -2723,7 +2727,7 @@ def readGrid(fileName):
                     ) = libcgns_utils.utils.getbcdatasetinfo(inFile, iBlock, iBoco, iBocoDataSet)
                     bcDSet = BocoDataSet(bocoDatasetName, bocoType)
 
-                    def getBocoDataSetArray(flagDirNeu):
+                    def getBocoDataSetArray(flagDirNeu, iDir):
```

Comment:
Just to clarify, this was a bug right? But luckily because the only places where we call `getBocoDataSetArray` the variable `iDir` was used in a loop so it was set correctly, meaning the answer was unchanged?

Reply:
Yes, that's correct for the Dirichlet BCs. The Neumann BCs though I think were actually broken because the variable was `nDir` instead of `iDir`. @eirikurj caught a related bug in #22